README.md

```diff
@@ -1,2 +1,14 @@
-# nerdy-home-theater
-nerdy way to watch tv built by you.
+## 💻  Nerdy-home-theater 📻
+    This is a project for those who wish to convert their old Pcs and unused Resources to functional high-tech interactive systems.
+    It will undergo numerous changes from good to terrible to :green_heart: Great.
+
+#### on-completion, it should:
+    [ ] Support viewing of videos from th comfort of your home.
+    [ ] Support viewing of pictures on high resolution.
+    [ ] Allow remote connection and control of the system.
+    [ ] Support multiple user experiences based on profiles.
+    [ ] Save user history and preferences.
+    [ ] Allow dectection of different Os mods and adjust settings accordingly.
+
```

Comment:
outlined a few tasks to accomplish  and get the project going in the right direction